README.md

```diff
@@ -2,23 +2,15 @@
 
 ![alt tag](https://raw.githubusercontent.com/tripal/tripal/7.x-3.x/tripal/theme/images/tripal_logo.png)
 
-Tripal is a toolkit for construction of online biological (genetics, genomics,
-breeding, etc), community database, and is a member of the 
-[GMOD](http://www.gmod.org) family of tools. Tripal v3 provides by default
-integration with the [GMOD Chado database](http://gmod.org/wiki/Chado_-_Getting_Started).
-Tripal's primary goals are: 
+Tripal is a toolkit for constructing online biological (genetics, genomics, breeding, etc.) community database, and Tripal is a member of the [GMOD](http://www.gmod.org) family of tools. **Tripal v3** provides integration with the [GMOD Chado database](http://gmod.org/wiki/Chado_-_Getting_Started) by default.
```

Comment:
>Tripal is a toolkit for constructing online biological (genetics, genomics, breeding, etc.) community database,

should probably be 

>Tripal is a toolkit for constructing online biological (genetics, genomics, breeding, etc.) community databases,

Reply:
Thank you, @bradfordcondon ! I have updated it.